etc/taskcluster/decision_task.py

```diff
@@ -0,0 +1,298 @@
+            "/repo/target/armv7-linux-androideabi/release/servoapp.apk",
+            "/repo/target/armv7-linux-androideabi/release/servoview.aar",
+        )
+        .find_or_create("build.android_armv7_release." + CONFIG.git_sha)
+    )
+
+
+def windows_dev():
+    return (
+        windows_build_task("Windows x64: dev build + unit tests")
+        .with_script(
+            # Not necessary as this would be done at the start of `build`,
+            # but this allows timing it separately.
+            "mach fetch",
+
+            "mach build --dev",
+            "mach test-unit",
+            "mach package --dev",
+        )
+        .with_artifacts("repo/target/debug/msi/Servo.exe",
+                        "repo/target/debug/msi/Servo.zip")
+        .find_or_create("build.windows_x64_dev." + CONFIG.git_sha)
+    )
+
+
+def windows_release():
+    return (
+        windows_build_task("Windows x64: release build")
+        .with_script("mach build --release",
+                     "mach package --release")
+        .with_artifacts("repo/target/release/msi/Servo.exe",
+                        "repo/target/release/msi/Servo.zip")
+        .find_or_create("build.windows_x64_release." + CONFIG.git_sha)
+    )
+
+
+def linux_wpt():
+    release_build_task = linux_release_build()
+    total_chunks = 2
+    for i in range(total_chunks):
+        this_chunk = i + 1
+        wpt_chunk(release_build_task, total_chunks, this_chunk, extra=(this_chunk == 1))
+
+
+def linux_release_build():
+    return (
+        linux_build_task("Linux x64: release build")
+        .with_script("""
+            ./mach build --release --with-debug-assertions -p servo
+            ./etc/ci/lockfile_changed.sh
+            tar -czf /target.tar.gz \
+                target/release/servo \
+                target/release/build/osmesa-src-*/output \
+                target/release/build/osmesa-src-*/out/lib/gallium
+        """)
+        .with_artifacts("/target.tar.gz")
+        .find_or_create("build.linux_x64_release." + CONFIG.git_sha)
+    )
+
+
+def wpt_chunk(release_build_task, total_chunks, this_chunk, extra):
+    name = "Linux x64: WPT chunk %s / %s" % (this_chunk, total_chunks)
+    script = """
+        ./mach test-wpt \
+            --release \
+            --processes 24 \
+            --total-chunks "$TOTAL_CHUNKS" \
+            --this-chunk "$THIS_CHUNK" \
+            --log-raw test-wpt.log \
+            --log-errorsummary wpt-errorsummary.log \
+            --always-succeed
+        ./mach filter-intermittents\
+            wpt-errorsummary.log \
+            --log-intermittents intermittents.log \
+            --log-filteredsummary filtered-wpt-errorsummary.log \
+            --tracker-api default
+    """
+    # FIXME: --reporter-api default
+    # IndexError: list index out of range
+    # File "/repo/python/servo/testing_commands.py", line 533, in filter_intermittents
+    #   pull_request = int(last_merge.split(' ')[4][1:])
+    if extra:
```

Comment:
Right. A previous version of this code used `extra` multiple times, so naming it and defining it once was nice. But now it might as well be inlined. Done.